doc/source/ray-core/ray-dashboard.rst

```diff
@@ -320,8 +594,8 @@ Actors
   * - **CPU Flame Graph**
     - Get a CPU flame graph for the specified actor. Refer to :ref:`dashboard-profiling` for more information.
 
-Logs
-~~~~
-The top level page for the logs view shows the list of nodes in the cluster. After clicking into a node, you can now see a list of all log files.
-
-Details of the different log files can be found here: :ref:`ray-logging`.
+Resources
+---------
+- `Ray Summit observability talk <https://www.youtube.com/watch?v=v_JzurOkdVQ>`_
+- `Ray metrics blog <https://www.anyscale.com/blog/monitoring-and-debugging-ray-workloads-ray-metrics>`_
+- `Ray dashboard roadmap <https://github.com/ray-project/ray/issues/30097#issuecomment-1445756658>`_
```

Comment:
Can we add the Observability Training Module here too? https://github.com/ray-project/ray-educational-materials/blob/main/Observability/Ray_observability_part_1.ipynb